hips/draw/tests/test_healpix.py

```diff
@@ -36,3 +37,10 @@ def test_healpix_to_hips(tmpdir, file_format):
 
     properties = (tmpdir / 'properties').read_text(encoding=None)
     assert file_format in properties
+
+def test_hips_to_healpix(tmpdir):
```

Comment:
I would prefer a simple test that doesn't use real data for two reasons:

- Hitting the network and disk is slow, and for the network error-prone
- Your test should end with an assert value on one or a few pixel values; if you take real data, it's not clear from looking at the test if the code works, you get an arbitrary value. If you set up a test case where you see what goes in from reading the test code, it's clear what should come out.